php/tests/PreviouslyGeneratedClassTest.php

```diff
@@ -12,7 +12,11 @@ class PreviouslyGeneratedClassTest extends TestBase
     public function testPrefixForReservedWords()
     {
         // In newer versions of PHP, we cannot reference the old class name.
-        if (version_compare(phpversion(), '8.1.0', '>=')) return;
+        if (version_compare(phpversion(), '8.1.0', '>=')) {
```

Comment:
If you look at composer, lower versions of PHP are not supported (the minumum supported version is PHP 8.1). So why not just get rid of this statement altogether?

Reply:
Great point..

I think that means we can also upgrade to phpunit 10.